nucleus/rna/models.py

```diff
@@ -17,6 +19,15 @@ def all_as_list(self):
         """Return all releases as a list of dicts"""
         return [r.to_dict() for r in self.prefetch_related('note_set').all()]
 
+    def recently_modified(self, days_ago=7, mod_date=None):
+        if mod_date is None:
+            mod_date = now() - timedelta(days=days_ago)
+
+        query = self.filter(Q(modified__gt=mod_date) |
```

Comment:
i'd go for `gte` to cover the edge case but it also depends on how you want to consume this